content/ytt/docs/latest/lang-ref-ytt-schema.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Schema module
+---
+
+## Overview
+
+`ytt`'s Schema feature provides a way to verify YAML structures and values with the help of annotations.
+Schema currently allows you to provide default data values, and verify that each data value file provided is compatible with the schema.
+
+The schema definition spells out the set of nodes that must make up that kind of YAML document. Documents that are type checked and validated against the schema are guaranteed to be proper.
+
+For example, in the case of Data Values, the schema guarantees to templates that all values exist and are of the correct type. The aleviates templates from doing existence and type checks, themselves.
```

Comment:
For this release of the Schema feature, I think it keeps the message more concise/clear if we describe it as being _all about_ Data Values. It's gonna be some time until we support checking a document via schema, generally.